src/Microsoft.DocAsCode.Build.Engine/HostService.cs

```diff
@@ -224,6 +224,7 @@ public MarkupResult ParseHtml(string html, FileAndType ft)
                     sources.Add(new LinkSourceInfo
                     {
                         Target = file,
+                        Anchor = anchor,
```

Comment:
>anchor [](start = 33, length = 6)

Note whether we need to remove anchor info in RemoveDebugInfo, which just added by Zhenghui?